### PR TITLE
installs a deploy_backup_tar script on the cds.

### PR DESCRIPTION
**Pull request by troy2914**

### CODE REVIEW

File: roles/cds_for_icpc/files/deploy_backup_tar [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+if isfile(backup_file):
+    path = "/srv/cds/contests/cds/"
+    destdir = join(realpath(path), "teams")
+    if not isdir(destdir):
+        mkdir(destdir)
+    with tarfile.open(backup_file, "r") as tar:
+        tar.extractall(destdir)
+        print("extracted {} to {}".format(backup_file, destdir))
+else:
+    print("ERROR: {} does not exist".format(backup_file))
```
Comment (vmcj) [inline roles/cds_for_icpc/files/deploy_backup_tar:18]: Assuming a reasonable version of python3 you can now do this
```suggestion
    print(f"ERROR: {backup_file} does not exist")
```

also on line 16, I find it easier to read but I'm not sure what is your opinion

File: roles/cds_for_icpc/files/deploy_backup_tar [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+from os import listdir, chdir, mkdir
+from os.path import isfile, islink, getmtime, join, realpath, isdir
+import tarfile
+
+backup_file = "backups.tar"
```
Comment (vmcj) [inline roles/cds_for_icpc/files/deploy_backup_tar:7]: If you make an ansible template out of this file you can directly pass the value for this.

**Review by vmcj — APPROVED** [review]
Should work on itself, 2 minor things which I would personally like.

File: roles/cds_for_icpc/files/deploy_backup_tar (hunk 1 above)
Comment (troy2914) [reply roles/cds_for_icpc/files/deploy_backup_tar]: ah yes,  this script was originally used on python3.6.   I have changed it to use f-strings.

File: roles/cds_for_icpc/files/deploy_backup_tar (hunk 2 above)
Comment (troy2914) [reply roles/cds_for_icpc/files/deploy_backup_tar]: It is a constant,  I have changed it to an uppercase variable, and added documentation.

**Pull request merged**